wordpress-mobile/gutenberg-mobile
# Fix toolbar bottom inset for iPhone X devices

Fixes https://github.com/WordPress/gutenberg/pull/16677#issuecomment-519089648

To test:
- Try booting the demo app on an iPhone 6 and an iPhone X (or Xs)
- Notice the problem in the aforementioned comment is gone

Update release notes:

^ Should not be a problem since the bug was never merged to `master`.
